src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -1013,6 +1018,20 @@ def load_arguments(self, _):
     with self.argument_context('staticwebapp create') as c:
         c.argument('location', arg_type=get_location_type(self.cli_ctx))
         c.argument('tags', arg_type=tags_type)
+        c.argument('sku', arg_type=static_web_app_sku_arg_type)
+        c.argument('app_location', options_list=['--app-location'],
+                   help="Location of your application code. For example, '/' represents the root of your app, "
+                        "while '/app' represents a directory called 'app'")
+        c.argument('api_location', options_list=['--api-location'],
+                   help="Location of your Azure Functions code. For example, '/api' represents a folder called 'api'.")
+        c.argument('app_artifact_location', options_list=['--app-artifact-location'],
```

Comment:
lets avoid including this and use output_location instead

Reply:
revised